From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 22 Oct 2020 19:09:35 +0000
Subject: telepathyClient: Promisify the right class

send_message_async() is a method of TpTextChannel, not the generic
TpChannel parent class.

(cherry picked from commit f5d1a2858d2a54f0344648ead219c553cd482737)

Bug: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3226
Origin: upstream, 3.38.2
---
 js/ui/components/telepathyClient.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/components/telepathyClient.js b/js/ui/components/telepathyClient.js
index b51bedc..0c9514e 100644
--- a/js/ui/components/telepathyClient.js
+++ b/js/ui/components/telepathyClient.js
@@ -9,7 +9,7 @@ try {
     ({ TelepathyGLib: Tp, TelepathyLogger: Tpl } = imports.gi);
 
     Gio._promisify(Tp.Channel.prototype, 'close_async', 'close_finish');
-    Gio._promisify(Tp.Channel.prototype,
+    Gio._promisify(Tp.TextChannel.prototype,
         'send_message_async', 'send_message_finish');
     Gio._promisify(Tp.ChannelDispatchOperation.prototype,
         'claim_with_async', 'claim_with_finish');
